Hazel/src/Hazel/Debug/Instrumentor.h

```diff
@@ -9,6 +9,38 @@
 
 namespace Hazel {
 
+	template <size_t N>
+	struct CleanExpressionResult
+	{
+		char Data[N];
+	};
```

Comment:
Wouldn't `std::array` work for this as well?

Reply:
It would, but a raw char array will work just fine

Reply:
yes - raw char array will do the trick, too